core/primitives/impl/scale_codec_impl.hpp

```diff
@@ -0,0 +1,69 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef KAGOME_PRIMITIVES_SCALE_CODEC_IMPL_HPP
+#define KAGOME_PRIMITIVES_SCALE_CODEC_IMPL_HPP
+
+#include "primitives/scale_codec.hpp"
+#include "scale/types.hpp"
+
+using namespace kagome::common::scale;
```

Comment:
`using namespace` in .hpp is not a good idea, as I know

Reply:
I will consider moving scale_codec to another namespace.